src/main/java/fi/metatavu/metamind/bot/MetamindBotConfiguration.java

```diff
@@ -123,8 +131,14 @@ private void addIntentSlots(Intent intent, AbstractIntentConfig intentConfig) {
     } 
   }
   
-  private OpenNLPIntentMatcher getIntentMatcher(IntentModel intentModel, OpenNLPTokenizer tokenizer, OpenNLPSlotMatcher slotMatcher) {
-    return new OpenNLPIntentMatcher(intentModel, tokenizer, slotMatcher, 0.75f, -1);
+  private List<OpenNLPIntentMatcher> getIntentMatchers(List<IntentModel> intentModels, OpenNLPTokenizer tokenizer, OpenNLPSlotMatcher slotMatcher) {
+    List<OpenNLPIntentMatcher> openNLPIntentMatchers = new ArrayList<OpenNLPIntentMatcher>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)